packages/fx-core/templates/plugins/resource/simpleauth/bicep/output.template.bicep

```diff
@@ -0,0 +1,3 @@
+
+output simpleAuth_skuName string = simpleAuthProvision.outputs.skuName
```

Comment:
can you confirm whether we need the sku output?

Reply:
Confirmed with @MuyangAmigo to keep skuName as output since cost-related info is vital to users.